docs/configuration-parameters.md

```diff
@@ -140,7 +140,7 @@ This section provides the details of all the configurations required for CarbonD
 | carbon.enableMinMax | true | Min max is feature added to enhance query performance. To disable this feature, set it false. |
 | carbon.dynamicallocation.schedulertimeout | 5 | Specifies the maximum time (unit in seconds) the scheduler can wait for executor to be active. Minimum value is 5 sec and maximum value is 15 sec. |
 | carbon.scheduler.minregisteredresourcesratio | 0.8 | Specifies the minimum resource (executor) ratio needed for starting the block distribution. The default value is 0.8, which indicates 80% of the requested resource is allocated for starting block distribution.  The minimum value is 0.1 min and the maximum value is 1.0. | 
-| carbon.search.enabled | false | If set to true, it will use CarbonReader to do distributed scan directly instead of using compute framework like spark, thus avoiding limitation of compute framework like SQL optimizer and task scheduling overhead. |
+| carbon.search.enabled (Alpha Feature) | false | If set to true, it will use CarbonReader to do distributed scan directly instead of using compute framework like spark, thus avoiding limitation of compute framework like SQL optimizer and task scheduling overhead. |
 
 * **Global Dictionary Configurations**
```

Comment:
In Local Dictionary section the following updates needs to be done.
1)      Remove the line: “44ad8fb40… Updated documentation on Local Dictionary Supoort |” in Page no: 7 at the Local Dictionary Configuration section in the Opensource PDF.

2)      Change  the description for “Local dictionary threshold” from: “The maximum cardinality for local dictionary generation (maximum - 100000)” 
to 
“The maximum cardinality for local dictionary generation (maximum value is 100000 and minimum value is 1000. If the “local_dictionary_threshold” value is set below 1000 or above 100000, then it would take the default value 10000)”

Reply:
The minimum value need not be mentioned now